mastadonApp/build.gradle.kts

```diff
@@ -1,8 +1,11 @@
+
+@Suppress("DSL_SCOPE_VIOLATION") // Remove when fixed https://youtrack.jetbrains.com/issue/KTIJ-19369
 plugins {
-  id("com.android.application")
-  kotlin("android")
+  alias(libs.plugins.android.application)
+  alias(libs.plugins.kotlin.android)
+  //alias(libs.plugins.kotlin.kapt)
+  //alias(libs.plugins.ksp)
 }
-
 android {
   namespace = "com.matrix159.mastadonclone"
   compileSdk = 33
```

Comment:
@oneSIX Thoughts on also putting these in the TOML?

Reply:
For these constants I guess they're okay in this app module - I don't foresee there being more than one Android target?